### PR TITLE
🔖 Prepare release 0.28.0

### PR DESCRIPTION
**Pull request by Viicos**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -21,11 +21,11 @@
 # -- Project information -----------------------------------------------------
 
 project = "ZGW Consumers"
-copyright = "2022, Maykin Media"
+copyright = "2023, Maykin Media"
```
Comment (sergei-maertens) [inline docs/conf.py:24]: copyrights do not need to be changed, it's just the starting point.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -11,7 +11,8 @@ Changes
   in the next major version.
 * Bump the minimum supported Python version to ``3.10``.
 * Added a ``ServiceFactory`` in the ``zgw_consumers.test.factories`` module.
-* Some depedencies that were installed by default are now moved to the ``testutils`` extra depedency group
+* Some depedencies that were installed by default are now moved to the ``testutils`` extra depedency group/
```
Comment (sergei-maertens) [inline CHANGELOG.rst:14]:
```suggestion
* Some dependencies that were installed by default are now moved to the ``testutils`` extra depedency group/
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

**Comment by sergei-maertens** [comment]
I'll make sure to wrap this up

File: docs/conf.py (hunk 1 above)
Comment (Viicos) [reply docs/conf.py:24]: Looking online I see a lot of different answers, and some people also recommended not to include any date at all for websites. Anyway, I don't know much about the legal aspect of copyrights and I even wonder how it relates to the project's license

**Pull request merged**